src/EnergyPlus/DataSurfaces.cc

```diff
@@ -86,6 +94,14 @@ namespace DataSurfaces {
 	using namespace DataPrecisionGlobals;
 	using namespace DataVectorTypes;
 	using namespace DataBSDFWindow;
+  using namespace DataHeatBalance;
```

Comment:
Will do. Thanks again. Really appreciating all these comments. I was looking here at LBNL for someone to review my code, but everyone is busy. I also realized extensive use of shared_ptr (in WCE) while in most cases passing by reference would suffice.